client/src/components/Payment/Payment.tsx

```diff
@@ -53,7 +53,7 @@ export default function Payment(): JSX.Element {
             <Grid container justifyContent="center">
                 <form className={classes.form} onSubmit={handleSubmit}>
                     <Grid container className={classes.formGrid}>
-                        <Grid className={classes.paper}>
+                        <Box className={classes.paper}>
```

Comment:
You have to specify it as a property for the Box component.

Eg: `<Box display="flex" alignItems="center"><Box>`

Reply:
put it in the spot needed, will likely create another issue for styling those card elements.